dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/view/AppsInterpolationTest.java

```diff
@@ -0,0 +1,112 @@
+package com.dotcms.rest.api.v1.apps.view;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.datagen.AppDescriptorDataGen;
+import com.dotcms.rest.api.v1.DotObjectMapperProvider;
+import com.dotcms.security.apps.AppDescriptor;
+import com.dotcms.security.apps.ParamDescriptor;
+import com.dotcms.security.apps.Secret;
+import com.dotcms.security.apps.Type;
+import com.dotcms.util.IntegrationTestInitService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AppsInterpolationTest {
+
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6392890)